apps/login/ui/UserAvatar.tsx

```diff
@@ -14,20 +14,20 @@ export default function UserAvatar({
   showDropdown,
 }: Props) {
   return (
-    <div className="flex h-full w-full flex-row items-center rounded-full border p-[1px] dark:border-white/20">
+    <div className="flex h-full flex-row items-center rounded-full border p-[1px] dark:border-white/20">
       <div>
         <Avatar
           size="small"
-          name={displayName ?? loginName}
-          loginName={loginName}
+          name={displayName ?? loginName ?? "A"}
+          loginName={loginName ?? "A"}
```

Comment:
What about `?` instead of `A`?

Reply:
? looks like an error to me.. I check if it can be rendered without a char

Reply:
removed the label entirely with emptystring